cirq/ops/three_qubit_gates_test.py

```diff
@@ -26,7 +26,7 @@
 )
 def test_eigen_gates_consistent_protocols(eigen_gate_type):
     cirq.testing.assert_eigengate_implements_consistent_protocols(
-            eigen_gate_type)
+            eigen_gate_type, is_global_phase_sensitive = True)
```

Comment:
Isn't this the default now? Don't specify it.

Reply:
The default is False.